setup.cfg

```diff
@@ -0,0 +1,81 @@
+[metadata]
+name = expertsystem
+version = 0.1.2-alpha
+author = The ComPWA Team
+author_email = compwa-admin@ep1.rub.de
+maintainer_email = compwa-admin@ep1.rub.de
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/ComPWA/expertsystem
+license = GPLv3 or later
+keywords =
+    HEP
+    PWA
+    particles
+    physics
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Intended Audience :: Education
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Physics
+    Typing :: Typed
+
+[options]
+python_requires = >=3.6
```

Comment:
should we also add some pip requirement?

Reply:
I guess technically we should (not sure how) but then we need to know more about what we expect of pip

Reply:
I can't think of any problems so far and it seems to work with pip 9.0.1. We can add it if we experience any problems